File: app/src/main/java/com/example/weatherchallengemvvmkotlin/util/WeatherFunctions.java
package com.example.weatherchallengemvvmkotlin.util;

public class WeatherFunctions {
    public static String convertKelvinToFahrenheit(Double kelvin) {
        return Long.toString(Math.round((9 / 5) * (kelvin - 273) + 32));
    }
}

File: app/src/main/java/com/example/weatherchallengemvvmkotlin/util/WeatherFunctions.kt
package com.example.weatherchallengemvvmkotlin.util

object WeatherFunctions {
    fun convertKelvinToFahrenheit(kelvin: Double): String {
        return java.lang.Long.toString(Math.round(9 / 5 * (kelvin - 273) + 32))
    }
}